### PR TITLE
Migrate to sbt 1.x

### LINKED ISSUE #123: Upgrade project to sbt 1.0
Bumping the sbt version will allow using newer AkkaHTTp instead of 

`libraryDependencies += "com.typesafe.akka" % "akka-http-experimental_2.10" % "2.0.4"`

### PR DESCRIPTION
**Pull request by ignasi35**
Fixes #123

I haven't done any attempt at improving the code (especially `SimpleHTTPServer`). We can address that on separate PRs.

### CODE REVIEW

**Comment by ignasi35** [comment]
Special mention to @muuki88 for the breadcrumbs he left on https://github.com/sbt/sbt-native-packager/pull/1000 wrt `PathFinder`. Very helpful :-)

File: build.sbt [hunk 1]
```diff
@@ -88,9 +90,9 @@ generateHtml <<= Def.taskDyn {
       blogDir,
       assetFingerPrint
     ).mkString(" ", " ", "")).value
-    outputDir.***.filter(_.isFile).get
+    (PathFinder(outputDir) ** AllPassFilter).filter(_.isFile).get
```
Comment (dwijnand) [inline build.sbt]:
```suggestion
    outputDir.allPaths.filter(_.isFile).get
```

File: build.sbt [hunk 2]
```diff
@@ -115,11 +117,15 @@ StylusKeys.compress := true
 
 pipelineStages := Seq(uglify, concat)
 WebKeys.pipeline ++= {
-  generateHtml.value pair relativeTo((target in generateHtml).value)
+  generateHtml.value pair (file => IO.relativize((target in generateHtml).value, file))
```
Comment (dwijnand) [inline build.sbt]:
```suggestion
  generateHtml.value pair Path.relativeTo((target in generateHtml).value)
```

File: build.sbt [hunk 3]
```diff
@@ -115,11 +117,15 @@ StylusKeys.compress := true
...
 }
+//     (PathFinder(outputDir) ** AllPassFilter).filter(_.isFile).get
```
Comment (dwijnand) [inline build.sbt]:
```suggestion
```

File: build.sbt [hunk 4]
```diff
@@ -115,11 +117,15 @@ StylusKeys.compress := true
...
 watchSources ++= {
-  ((sourceDirectory in Compile).value / "markdown").***.get ++
-    (sourceDirectory.value / "blog").***.get
+  val markdownFolder: File = (sourceDirectory in Compile).value / "markdown"
+  val blogFolder: File = sourceDirectory.value / "blog"
+  val markdown: Seq[File] = (PathFinder(markdownFolder) ** AllPassFilter).get
```
Comment (dwijnand) [inline build.sbt]:
```suggestion
  val markdown: Seq[File] = markdownFolder.allPaths.get
```

File: build.sbt [hunk 5]
```diff
@@ -115,11 +117,15 @@ StylusKeys.compress := true
...
+  val blog: Seq[File] = (PathFinder(blogFolder) ** AllPassFilter).get
```
Comment (dwijnand) [inline build.sbt]:
```suggestion
  val blog: Seq[File] = blogFolder.allPaths.get
```

**Comment by ignasi35** [comment]
Thanks @dwijnand for the suggestions! :-)

**Pull request merged**